services/graphql-server/src/graphql/definitions/platform/content/index.js

```diff
@@ -485,6 +485,8 @@ input AllPublishedContentQueryInput {
   sectionId: Int
   "Deprecated. Use includeContentTypes instead."
   contentTypes: [ContentType!] = []
+  "Limit results to items matching specific custom attribute key value pairs"
+  customAttributes: [ContentCustomAttributeQueryInput] = []
```

Comment:
The custom attribute input (inside the array) should be non-nullable, i.e.
`[ContentCustomAttributeQueryInput!] = []`